crates/cargo-bsp/src/cargo_communication/request_actor.rs

```diff
@@ -82,13 +84,12 @@ where
     fn handle_cargo_event(&mut self, message: CargoMessage) {
         // handle information and create notification based on that
         match message {
-            CargoMessage::CargoStdout(msg) => self.handle_cargo_information(msg),
+            CargoMessage::CargoStdout(msg) => self.deserialize_and_handle_cargo_information(msg),
             CargoMessage::CargoStderr(msg) => self.log_message(MessageType::Error, msg, None),
         }
     }
 
     pub fn run(mut self) {
-        self.report_task_start(self.state.root_task_id.clone(), None, None);
```

Comment:
dlaczego już nie wysyłamy tego info?

Reply:
Bo root task jest zaczynany w run_unit_graph (to jest task dla całego requestu)

Reply:
hm... to może wgl wydzielić wysyłanie tego root taska do funkcji osobnej wołanej jeszcze zanim będzie się robić `run_unit_graph`?

Reply:
Wydzielone